cribl_stream/manifest.json

```diff
@@ -0,0 +1,59 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "2ef15aea-af91-4769-940c-2b124e4d04a6",
+  "app_id": "cribl-stream",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect observability data in a vendor-neutral data telemetry pipeline",
+    "title": "Cribl Stream",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Cribl Stream Logo",
+        "image_url": "images/CriblStream.FULL_COLOR.png"
+      },
```

Comment:
The logo will automatically be added to your tile, and we do not recommend adding it to the media carousel. Instead, could you please upload a screenshot of your dashboard? Please note that all media carousel images must follow our [guidelines](https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#media-carousel), notably the 16:9 aspect ratio rule.

Reply:
Just uploaded a screen capture of the Cribl Stream Dashboard for Datadog : cribl_stream/images/cribl_data_dog_dashboard.png .